packages/EasyActivity/.gitattributes

```diff
@@ -0,0 +1,3 @@
+docs/ export-ignore
```

Comment:
I believe we should leave `docs`, as these files are mentioned in the `readme.md`

Or add `readme.md` for ignoring.

Wdyt?

Reply:
Agree. Ignored `readme.md`

Reply:
Thanks.